assets/js/pages/Users/Users.jsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Button from '@common/Button';
+import Table from '@common/Table';
+import PageHeader from '@common/PageHeader';
+import Modal from '@common/Modal';
+import Tooltip from '@common/Tooltip';
+import Banner from '@common/Banners/Banner';
+import { EOS_LOADING_ANIMATED } from 'eos-icons-react';
+
+const USER_CREATE_ROUTE = '/users/new';
+
+function Users({
+  handleDeleteUser = () => {},
+  navigate = () => {},
+  setModalOpen = () => {},
+  setDeleteUserId = () => {},
+  deleteUserId = 0,
+  modalOpen = false,
+  users = [],
```

Comment:
As far as I remember, we don't usually put an empty list as is as default.
There was some error with it.
We just define a constant on top (`defaultUsers`) and assign it.
I don't remember why, but we do it everywhere